numba_dpex/core/caching.py

```diff
@@ -0,0 +1,387 @@
+# Copyright 2022 Intel Corporation
```

Comment:
Copyright header should follow style of rest of the files. We use `reuse` to generate the headers.

Reply:
How do I use `reuse` to generate a correct header? I couldn't find any pointers in the repo.

Reply:
> Copyright header should follow style of rest of the files. We use `reuse` to generate the headers.

How do I use `reuse` to generate a correct header? I couldn't find any pointers in the repo.